Dockerfile

```diff
@@ -3,7 +3,7 @@
 
 ## prep the base image.
 #
-FROM golang:1.16.2-buster as base
+FROM golang:1.16.15-bullseye as base
```

Comment:
I made this change because some of the newer binaries didn't play well with the now-outdated debian buster (some c++ shared libraries outdated).

Reply:
Maybe it's time to upgrade Go version to v0.18.0 now? We recently upgraded required Go version for the CLI.